dod-advana/gamechanger-web
# Task/uot 126657

## Description
Makes the Advana pill on decoupled redirect to advana.data.mil rather than opening a login popup.

## Tickets
UOT-126657

## Testing Instructions
Click on the Advana pill in the upper right hand corner on various screens (home, global search home, details page, admin page) on decouple and ensure the page is redirected directly to advana.data.mil rather than opening a login popup.
